packages/form-js-editor/src/render/components/FieldResizer.js

```diff
@@ -0,0 +1,188 @@
+import {
+  useRef
+} from 'preact/hooks';
+
+import {
+  classes,
+  query as domQuery
+} from 'min-dom';
+
+import { useService } from '../hooks';
+
+import { createDragger, throttle } from './Util';
+
+import { DRAG_NO_MOVE_CLS } from '../../features/dragging/Dragging';
+
+import classNames from 'classnames';
+
+const COLUMNS_REGEX = /^cds--col(-lg)?/;
+
+const ELEMENT_RESIZING_CLS = 'fjs-element-resizing';
+
+const RESIZE_DRAG_PREVIEW_CLS = 'fjs-resize-drag-preview';
+
+const GRID_OFFSET_PX = 16;
+
+
+export function FieldResizer(props) {
+
+  const {
+    field,
+    position
+  } = props;
+
+  const ref = useRef(null);
+
+  const formLayoutValidator = useService('formLayoutValidator');
+  const modeling = useService('modeling');
+
+  // we can't use state as we need to
+  // manipulate this inside dragging events
+  const context = useRef({
+    startColumns: null,
+    newColumns: null
```

Comment:
Maybe we should use 0 instead of `null`? This way we keep the types as just `number` for this